test/ck_tile/grouped_gemm_quant/test_grouped_gemm_util_quant.hpp

```diff
@@ -292,6 +522,15 @@ class TestCkTileGroupedGemmQuant : public ::testing::Test
                 AQK = 1; // Row quantization: tensor shape [M, 1] or [1]
                 BQK = 1; // Column quantization: tensor shape [1, N] or [1]
             }
+            else if constexpr(QuantType == ck_tile::QuantType::AQuantGrouped)
+            {
+                AQK = K / 128; // Group quantization: AQK = K / GroupSize
+                BQK = 0;       // No B quantization
+                if(K % 128 != 0)
+                {
+                    throw std::runtime_error("K must be divisible by 128 for AQuantGrouped mode");
+                }
+            }
```

Comment:
Should `QuantGroupSize::kK` be used? (as in `example/ck_tile/17_grouped_gemm/quant_run_grouped_gemm_example.inc`)
Same for `else if constexpr(QuantType == ck_tile::QuantType::BQuantGrouped)` branch